test.md

```diff
@@ -0,0 +1 @@
+"This should be ignored by git"
```

Comment:
According to the requirements and as stated by the previous reviewer, please make sure that the test.md file is ignored by git. Ignored files shouldn't be present in your remote repository. 🤗